README.md

```diff
@@ -81,6 +81,52 @@ signing {
     sign(publishing.publications["mavenJava"])
 }
 ```
+
+#### Publishing with Ivy ####
+
+There are cases where it may be necessary to use the `ivy-publish` plugin instead of `maven-publish`.
+For example, when publishing Sbt plugins the directory structure needs to be customized which is only possible with Gradle's `IvyArtifactRepository`.
+
+In such cases, you need to apply the `ivy-publish` plugin and configure the `nexusPublishing` extension's `publicationType` to `IVY` (default is `MAVEN`).
+
+In case of Ivy publishing, because of compatibility with Sonatype the nexus repository layout will be used by default
+
+```groovy
+nexusPublishing {
+    publicationType = io.github.gradlenexus.publishplugin.NexusPublishExtension.PublicationType.IVY
+}
+```
+
+Or use the kotlin DSL:
+
+```kotlin
+nexusPublishing {
+    publicationType.set(io.github.gradlenexus.publishplugin.NexusPublishExtension.PublicationType.IVY)
+}
+```
+
+##### Using Ivy repositories with different artifact patterns ####
+
+In case of ivy it's possible to override the default artifact pattern that is used, which is the Maven pattern due to compatibility reasons with sonatype
+
+To change the pattern of artifacts and ivy files use
+
+```groovy
+ivyPatternLayout {
```

Comment:
I think we should show how to configure this on a repo here and below to make it clear where `ivyPatternLayout` needs to be called.